### DISCUSSION TITLE
升级到3.11后 阿里云依旧显示 failed get link: Device offline

### DISCUSSION THREAD

**Discussion by kesonglu** [AlistGo/alist#3410] [Q&A]
failed get link: Device offline
大半夜刚更新的最新版本 密钥也更新了一边 但是 报错依旧

**Comment by a888kay** [comment]
我也是≥﹏≤难受

**Comment by brostart** [comment]
临时解决办法就是挂载共享文件夹，我已经改好了，亲测可用。

**Comment by LoveU3th** [comment]
同样的问题，测试了共享挂载可以，自己的不行

**Reply by a888kay** [reply]
我的解决了，方案放在下面

**Comment by a888kay** [comment]
我解决了（｡ò ∀ ó｡）

我的解决方案：打开手机阿里app >登录设备
然后全部退出登录再获取新的token，到alist把新的token填上保存，再重启一下服务器就正常了

**Reply by ctt727** [reply]
> 我解决了（｡ò ∀ ó｡）
> 
> 我的解决方案：打开手机阿里app >登录设备 然后全部退出登录再获取新的token，到alist把新的token填上保存，再重启一下服务器就正常了

谢谢你，我用你的方法解决了。

**Reply by 14960** [reply]
此方法有效，感谢！

**Reply by aTenb** [reply]
无效

**Reply by word911** [reply]
记得弄完之后重启。否则不会显示。

在这里，通过点击、扫描二维码获得手机app的token
https://alist.nn.ci/zh/guide/drivers/aliyundrive.html#%E5%88%B7%E6%96%B0%E4%BB%A4%E7%89%8C

**Reply by hayhaps** [reply]
按解决方案操作无效

**Reply by jfdnet** [reply]
我把已登录设备(我已经有8个)全部退出；
没有更新token；
重启Alist；

然后就恢复使用了。

**Reply by dfvips** [reply]
操作无效+1

**Comment by GSANDLY** [comment]
由于阿里云盘 referer 的限制，必须使用移动端 token。 使用桌面 Web 令牌将导致无法下载和预览。 当然，如果你在本地使用或者带宽足够大，你也可以开启代理，让桌面 Web 的 refresh token 正常工作。

https://alist.nn.ci/zh/guide/drivers/aliyundrive.html

**Comment by oyksoft** [comment]
遇到同样的问题。已解决，一定要重启下alist !!!

**Reply by dfvips** [reply]
并不能解决，重启了alist也一样。而且试了手机的token了。但是一保存，alist就更新新的token。始终无法正常使用阿里云盘。
